### PR TITLE
Removes targetting mode

### PR DESCRIPTION
**Pull request by DominikPanic**

### CODE REVIEW

**Comment by Ghommie** [comment]
I know it's just easier to clip out the actions and buttons, but do remember to clean up the remaining left overs if you going to obliterate this baycode feature.

**Comment by DominikPanic** [comment]
I've looked at it and it's pretty deeply ingrained and linked to other features.

**Comment by Ghommie** [comment]
As expected...
I may say "make this a configurable feature" but highly doubt people would even care about this one feature, not even myself.

**Comment by Rohesie** [comment]
Do you plan on finishing the rooting-out of this feature?

**Comment by DominikPanic** [comment]
> 
> 
> Do you plan on finishing the rooting-out of this feature?

Should be removed fully now, RIP 600 lines of code.

**Comment by DominikPanic** [comment]
Reverted the full-removal as discussed, it should only delete the visual elements again.

File: code/modules/projectiles/updated_projectiles/gun_helpers.dm [hunk 1]
```diff
@@ -272,8 +272,7 @@ should be alright.
 
 /obj/item/weapon/gun/attack_self(mob/user)
 	..()
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/gun_helpers.dm]:
```suggestion
	. = ..()
```

**Pull request merged**